### PR TITLE
Image urls

### PR DESCRIPTION
**Pull request by DominykasSip**

### CODE REVIEW

**Review by JesusGonzalezA — APPROVED** [review]

File: backend/Domain/Entities/Food.cs [hunk 1]
```diff
@@ -8,13 +8,16 @@ public class Food : BaseEntity
...
         public string IdRestaurant { get; set; }
+        public string ImageURL { get; set; }
         public virtual IEnumerable<TypeOfFood> TypesOfFood { get; set; }
         public DateTime StartDecreasingAt { get; set; }
         public DecreaseType DecreaseType { get; set; }
         public double IntervalTimeInMinutes { get; set; }
         public decimal AmountPerInterval { get; set; }
         public double PercentPerInterval { get; set; }
 
+        private const string DEFAULT_IMAGE_URL = "https://genesisairway.com/wp-content/uploads/2019/05/no-image.jpg";
```
Comment (audrius-savickas) [inline backend/Domain/Entities/Food.cs:19]: this same URL is used in 3 files, I believe all of them should import from one constant

**Review by audrius-savickas — CHANGES_REQUESTED** [review]
you should also add urls to already existing json items

**Comment by DominykasSip** [comment]
> you should also add urls to already existing json items

I was thinking to do a seperate branch and redo the whole restaurant json because It's outdated and a bit random.

**Comment by DominykasSip** [comment]
And also Donatas merged json changes to main and this would cause conflicts

**Review by audrius-savickas — APPROVED** [review]

**Pull request merged**